camel/embeddings/open_source_embeddings.py

```diff
@@ -0,0 +1,64 @@
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+from typing import Any, List, Union
+
+from numpy import ndarray
+
+from camel.embeddings import BaseEmbedding
+
+
+class SentenceTransformerEmbedding(BaseEmbedding[str]):
+    r"""This class provides functionalities to generate embeddings
+    using a specified model from `Sentence Transformers`.
+
+    References:
+        https://www.sbert.net/
+    """
+
+    def __init__(self, model_name: str = 'intfloat/e5-large-v2'):
+        r"""Initializes the: obj: `SentenceTransformerEmbedding` class
+        with the specified transformer model.
+
+        Args:
+            model_name (str, optional): The name of the model to use.
```

Comment:
`model_name` is an optional argument, so I use (str, optional) rather than (str). is the suggested change aimed to align with our defined docstring format? If no, I will suggest let's keep `optional` there.